tests/IoTeX/StakingTests.cpp

```diff
@@ -277,15 +319,22 @@ TEST(TWIoTeXStaking, SignAll) {
         // signed action's hash
         ASSERT_EQ(hex(output.hash()),
                   "74b2e1d6a09ba5d1298fa422d5850991ae516865077282196295a38f93c78b85");
-        input.release_staketransferownership();
-        output.release_encoded();
-        output.release_hash();
+        //input.release_stakeadddeposit();
```

Comment:
intended change?

Reply:
Yes.  Same input object was reused in multiple test cases, and this detach was needed (I guess) to prevent multiple release --> so there was an unreleased sub-object.  Now no reuse is done, no trick is needed.
I should've removed these lines, I did now.